src/Api.Rest.Dtos.Tests/Data/AttributeDtoTest.cs

```diff
@@ -0,0 +1,300 @@
+﻿namespace Zeiss.PiWeb.Api.Rest.Dtos.Tests.Data
+
+{
+	#region usings
+
+	using System;
+	using FluentAssertions;
+	using NUnit.Framework;
+	using Zeiss.PiWeb.Api.Rest.Dtos.Data;
+
+	#endregion
+
+	[TestFixture]
+	public class AttributeDtoTest
```

Comment:
I would not mix FluentAssertions and NUnit Assert calls in the Test class. I thinks it's better to just use one type of assertion library.

Reply:
Agreed, I changed it to only use FluentAssertions.